the-collab-lab/tcl-54-smart-shopping-list
# Issue 23 | Create an Add Item card within the shopping list items view

## Description

`Add items` button is added inside the shopping list display when items are filled.

## Related Issue

Closes #50 

## Acceptance Criteria

- [x] Create add item button inside filled shopping list display
-
## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |

## Updates

### Before

![image](https://user-images.githubusercontent.com/89554728/222921927-22cced96-7c56-443c-823d-b5ea8fba3773.png)

### After

![image](https://user-images.githubusercontent.com/89554728/222921964-5d3dc71b-eb30-4984-a8fd-f0ca3fc8c3cf.png)


## Testing Steps / QA Criteria

1. In your terminal, run the command `git pull` and then `git checkout ad-svg-import-debug`
2. Then run `npm start`
3. Confirm there is an `Add items` button inside the filled shopping list display
